src/overlays/actors/ovl_Obj_Dhouse/z_obj_dhouse.h

```diff
@@ -7,11 +7,44 @@ struct ObjDhouse;
 
 typedef void (*ObjDhouseActionFunc)(struct ObjDhouse*, GlobalContext*);
 
+#define OBJDHOUSE_GET_7F(thisx) ((thisx)->params & 0x7F)
+
+typedef struct {
+    /* 0x00 */ Vec3f unk_00;
+    /* 0x0C */ Vec3f unk_0C;
+    /* 0x18 */ Vec3s unk_18;
+    /* 0x1E */ s8 unk_1E;
+} ObjDhouseStruct1; // size = 0x20
+
+typedef struct {
+    /* 0x00 */ Vec3f unk_00;
+    /* 0x0C */ Vec3f unk_0C;
+    /* 0x18 */ f32 unk_18;
+    /* 0x1C */ Vec3s unk_1C;
+    /* 0x22 */ s16 unk_22;
+    /* 0x24 */ s16 unk_24;
+    /* 0x26 */ s16 unk_26;
+    /* 0x28 */ s8 unk_28;
+    /* 0x29 */ u8 unk_29;
+} ObjDhouseStruct2; // size = 0x2C
+
+typedef struct {
+    /* 0x00 */ Gfx* unk_00;
+    /* 0x04 */ Gfx* unk_04;
+    /* 0x08 */ Vec3f unk_08;
+    /* 0x14 */ Vec3f unk_14;
+    /* 0x20 */ s16 unk_20;
+    /* 0x22 */ s16 unk_22;
+    /* 0x24 */ s16 unk_24;
+    /* 0x26 */ s8 unk_26;
+} ObjDhouseStruct3; // size = 0x28
```

Comment:
This one can go in the C file since it's not used in the header.

Reply:
It's used in forward-declared functions, it looked better to put it here rather than at the top before functions, and looked better than forward-declaring the struct.